week2/Week2_Day4_Exercises.Rmd

```diff
@@ -0,0 +1,157 @@
+---
+title: "Week 2 Day 4 Classwork"
+output: html_document
+date: '2022-06-09'
+---
+
+```{r}
+library(tidyverse)
+library(dplyr)
+```
+
+
+2.2 Heart transplants, Part II. Exercise 1.50 introduces the Stanford Heart Transplant Study. Of the 34 patients in the control group, 4 were alive at the end of the study. Of the 69 patients in the treatment group, 24 were alive. The contingency table below summarizes these results.
+
+      Group
+            Control Treatment Total
+      Alive     4     24        28 
+      Dead    30        45      75
+      Total     34    69        103

```

Comment:
Markdown has special formatting for tables!

```markdown
|Group|Control|Treatment|Total|
|-------|--------|-----------|-----|
|Alive|4|24|38|
|Dead|30|45|75|
|Total|34|69|103|
```
|Group|Control|Treatment|Total|
|-------|--------|-----------|-----|
|Alive|4|24|38|
|Dead|30|45|75|
|Total|34|69|103|